.github/workflows/codeql.yml

```
@@ -1,99 +1,54 @@
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
 name: "CodeQL Advanced"
 
 on:
   push:
-    branches: [ "master" ]
+    branches: [ "master", "dev" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)